src/example/example.py

```diff
@@ -14,12 +14,16 @@
 """
 
 import logging
+import os
 import sys
 
 import docopt
+import pkg_resources
```

Comment:
Where does this package come from?

Reply:
I believe that is pulled in with `pip`.   Let me check on your coverage thang first.

Reply:
`setuptools` 
https://setuptools.readthedocs.io/en/latest/pkg_resources.html

Reply:
I added `setuptools` as an explicit requirement now.